TodoList.js

```diff
@@ -0,0 +1,45 @@
+Vue.createApp({
+  data () {
+    return {
+      todos: []
+    }
+  },
+
+  created () {
+    if (this.$_getTodos()) {
+      this.todos = this.$_getTodos()
+    }
+  },
+
+  methods: {
+    create () {
+      this.todos.push({
+        id: Date.now().toString(36) + Math.random().toString(36),
+        title: this.title,
+        isEditing: false
+      })
+      this.$_save()
+      this.title = ''
+    },
+
+    toggleUpdateButton (todo) {
+      todo.isEditing = !todo.isEditing
+      this.$_save()
+    },
+
+    destroy (todo) {
+      this.todos = this.todos.filter(nowTodo => {
+        return nowTodo.id !== todo.id
+      })
```

Comment:
1 行で書いたほうが見やすいかなーと思います。

```suggestion
      this.todos = this.todos.filter(nowTodo => newTodo.id !== todo)
```